integration-tests/load/vrfv2plus/gun.go

```diff
@@ -0,0 +1,57 @@
+package loadvrfv2plus
+
+import (
+	"github.com/rs/zerolog"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions/vrfv2plus"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions/vrfv2plus/vrfv2plus_config"
+	"github.com/smartcontractkit/wasp"
+	"math/big"
+)
+
+/* SingleHashGun is a gun that constantly requests randomness for one feed  */
+
+type SingleHashGun struct {
+	contracts       *vrfv2plus.VRFV2_5Contracts
+	keyHash         [32]byte
+	subID           *big.Int
+	vrfv2PlusConfig vrfv2plus_config.VRFV2PlusConfig
+	logger          zerolog.Logger
+}
+
+func SingleFeedGun(
+	contracts *vrfv2plus.VRFV2_5Contracts,
+	keyHash [32]byte,
+	subID *big.Int,
+	vrfv2PlusConfig vrfv2plus_config.VRFV2PlusConfig,
+	logger zerolog.Logger,
+) *SingleHashGun {
+	return &SingleHashGun{
+		contracts:       contracts,
+		keyHash:         keyHash,
+		subID:           subID,
+		vrfv2PlusConfig: vrfv2PlusConfig,
+		logger:          logger,
+	}
+}
+
+// Call implements example gun call, assertions on response bodies should be done here
+func (m *SingleHashGun) Call(l *wasp.Generator) *wasp.CallResult {
+	//todo - should work with multiple consumers and consumers having different keyhashes and wallets
+
+	_, err := vrfv2plus.RequestRandomnessAndWaitForFulfillment(
```

Comment:
it can wait as well, afaik, each gun runs in separate go routine, so should not affect anything.

I just need to solve the issue when test stops and loki metrics stopped from being sent to Grafana, so then if we dont wait then Grafana metrics are not accurate, the fulfilment stats in particular